config/environments/development.rb

```diff
@@ -31,9 +31,10 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
```

Comment:
Comment should be changed to match new behavior

Reply:
will fix 👍

Reply:
outdated comment removed. 🌈